### ISSUE TITLE
scrubyt on 1.9.1

### ISSUE THREAD

**Issue by guiguidoc** [scrubber/scrubyt#6]
var/lib/gems/1.9.1/gems/scrubyt-0.4.06/lib/scrubyt.rb:1: warning: variable $KCODE is no longer effective; ignored
no such file to load -- jcode

**Comment by jakemoffatt** [comment]
I get this same error on 1.9.1 on Windows, looking for the answer.  Rubygems is def included, this is using the canonical google example.

I also have installed the RubyInstaller DevKit so that I could build Hpricot (before i installed Scrubyt) and Hpricot works for sure.
- Update*:
  Seems jcode is removed from 1.9.1, what now?

**Comment by papayaah** [comment]
anyone has a solution to this? im using ruby 1.9.2

**Comment by jakeonrails** [comment]
http://stackoverflow.com/questions/4120720/no-jcode-gem-when-running-rails-server

TL;DR:
find the line of code including jcode and do this:
    require 'jcode' if RUBY_VERSION < '1.9'

**Comment by papayaah** [comment]
i thnk i tried doing that but still get bunch of errors, but its okay, I went with a more modern library - nokogiri. I am good now, i just need to be able to parse an arbitrary HTML with powerful DOM parsing capabiity. Nokogiri offers xpath and css style selectors.

**Comment by TylerRick** [comment]
I get the same error. Please update scrubyt to support Ruby 1.9.
